CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 6.2.3 (2022-8-18)
+- Fix resource loading in GoogleSignInSwift with CocoaPods use_frameworks! ([#197](https://github.com/google/GoogleSignIn-iOS/pull/197))
+- Exclude GoogleSignInSwift for i386 in addition to armv7. ([#166](https://github.com/google/GoogleSignIn-iOS/pull/166))
+
+# 6.2.2 (2022-5-27)
```

Comment:
I think you didn't mean to duplicate this section.

Reply:
Ugh. Sorry about that.